documents/building/README.md

```diff
@@ -66,6 +66,11 @@ sudo apt-get install -yq --no-install-recommends \
 #required if the ENABLE_TESTING option is enabled
 sudo apt-get install -yq --no-install-recommends \
     libcpputest-dev
+
+#For testing gtest is also used. This is fetched using a cmake 3.11+ feature,
```

Comment:
Yes, already mentioned by Roy and updated to 3.13

Reply:
Was an old remark, I always forget to post the review. Kinda used to adding it, and expecting it to be available. Instead of having to actually submit it ;)